custom_model_runner/datarobot_drum/resource/components/Python/r_fit/fit.R

```diff
@@ -107,6 +107,8 @@ outer_fit <- function(output_dir, input_filename, target_filename,
 
     if (!is.null(positive_class_label) && !is.null(negative_class_label)){
         class_order <- c(negative_class_label, positive_class_label)
+    } else if (!is.null(class_labels)) {
+        class_order <- class_labels
```

Comment:
I don't see a reason not to combine the binary and multiclass cases everywhere. do you?

Reply:
what do you mean? they are. We use the target type to determine the labels to use, then use them.